api/SourceBuffer.json

```diff
@@ -761,7 +761,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/SourceBuffer/trackDefaults",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
This hasn't shipped, `'trackDefaults' in SourceBuffer.prototype` is false in Chrome.